apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/dashboard/_components/ConnectedDocumentsTable/index.tsx

```diff
@@ -136,7 +136,7 @@ export default function ConnectedDocumentsTable({
                   .detail({ id: document.projectId })
                   .commits.detail({ uuid: HEAD_COMMIT })
                   .documents.detail({ uuid: document.documentUuid })
-                  .evaluations.detail(document.evaluationUuid).root,
+                  .evaluations.detail(document.evaluationId).root,
```

Comment:
Aren't we finally using `evaluationUuid`?

Reply:
Explained offline and here https://github.com/latitude-dev/latitude-llm/pull/162#discussion_r1758317022